lib/src/presentation/view/card_list.dart

```diff
@@ -0,0 +1,341 @@
+                  color: Colors.black,
+                  child: const Text(
+                    kCardListDemons,
+                    textAlign: TextAlign.center,
+                    style: TextStyle(
+                      fontSize: cardTextInfoFontSize,
+                      fontWeight: FontWeight.bold,
+                    ),
+                  ),
+                ),
+                onTap: () {
+                  _currentUrl = _demonUrl;
+                  widget.cardBloc.sinkCardList(
+                    _currentUrl,
+                  );
+                  updateCurrentPage();
+                  Navigator.pop(context);
+                },
+              ),
+            ],
+          ),
+        ),
+        body: SingleChildScrollView(
+          child: Center(
+            child: DecoratedBox(
+              decoration: const BoxDecoration(
+                image: DecorationImage(
+                  image: AssetImage(
+                    assetPathBackgroundImage,
+                  ),
+                  fit: BoxFit.cover,
+                ),
+              ),
+              child: Column(
+                children: [
+                  Padding(
+                    padding: const EdgeInsets.only(
+                      left: cardTextPaddingLeftTopRight,
+                      top: cardTextPaddingLeftTopRight,
+                      right: cardTextPaddingLeftTopRight,
+                      bottom: cardTextPaddingBottom,
+                    ),
+                    child: Container(
+                      constraints: const BoxConstraints(
+                        minHeight: kCardListBoxMinHeight,
+                      ),
+                      child: StreamBuilder<List<HearthstoneCard>>(
+                        stream: widget.cardBloc.cardListStream,
+                        builder: (
+                          BuildContext context,
+                          AsyncSnapshot<List<HearthstoneCard>> snapshot,
+                        ) {
+                          if (snapshot.connectionState ==
+                              ConnectionState.waiting) {
+                            return const Center(
+                              child: CircularProgressIndicator(
+                                strokeWidth: kCardListProgressIndStrokeWidth,
+                                backgroundColor: Colors.black,
+                                color: Colors.red,
+                              ),
+                            );
+                          }
+                          if (widget.cardBloc.isLoading == true) {
+                            return const Center(
+                              child: CircularProgressIndicator(
+                                strokeWidth: kCardListProgressIndStrokeWidth,
+                                backgroundColor: Colors.black,
+                                color: Colors.red,
+                              ),
+                            );
+                          }
```

Comment:
What happened if you comment the if relative to snapshot.connectionState == ConnectionState.waiting? 
Do you still see the loaders?

Reply:
Done.